pom.xml

```diff
@@ -256,9 +262,18 @@ THE SOFTWARE.
     <dependency>
       <groupId>org.jenkins-ci.plugins.workflow</groupId>
       <artifactId>workflow-aggregator</artifactId>
-      <version>${workflow.version}</version>
-      <classifier>tests</classifier>
+      <version>${workflow-aggregator.version}</version>
       <scope>test</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>org.jenkins-ci.plugins.workflow</groupId>
+          <artifactId>workflow-step-api</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>org.jenkins-ci.plugins</groupId>
+          <artifactId>script-security</artifactId>
```

Comment:
One of the CI tests is failing with `java.lang.NoClassDefFoundError: org/jenkinsci/plugins/scriptsecurity/scripts/ScriptApproval`, so maybe these shouldn't be excluded?

Reply:
seems like, I have had some problems with maven-enforcer-plugin to correct the dependencies.